Demo/ViewControllers/DemoTableViewCell.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+import UIKit
+
+class DemoTableViewCell: UITableViewCell {
+
+    // MARK: - Init
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        textLabel?.font = .title3
+        textLabel?.textColor = .milk
+        backgroundColor = .clear
+
+        let view = UIView(frame: .zero)
+        selectedBackgroundView = view
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Public methods
+
+    func configure(with model: Row) {
```

Comment:
I would just configure with title if it's the only thing needed from `Row` model.

Reply:
Yep